gsrs-spring-boot-autoconfigure/src/main/java/gsrs/controller/AbstractImportSupportingGsrsEntityController.java

```diff
@@ -679,6 +679,34 @@ public ResponseEntity<Object> updateImport(@RequestBody JsonNode updatedJson,
         return new ResponseEntity<>(GsrsControllerUtil.enhanceWithView(itmd, queryParameters), HttpStatus.OK);
     }
 
+    @hasAdminRole
+    @PostGsrsRestApiMapping(value = {"/stagingArea"})
+    public ResponseEntity<Object> createEntity(@RequestBody String newEntity,
+                                               @RequestParam Map<String, String> queryParameters) throws Exception {
+        log.trace("in createEntity");
+        String entityType = queryParameters.get("entityType");//type of domain object to create
```

Comment:
Won't this be a POST at a level like "/api/v1/substances/stagingArea", does the client really need to tell you it's for substances again?

Reply:
That's a good point.
We have the same requirement for the /import endpoint and I copied it.
The other issue: how do we map from 'substances' to 'ix.ginas.models.v1.Substance?'